wikimedia/wikipedia-ios
# Image Recommendations - Add empty state

**Phabricator:** https://phabricator.wikimedia.org/T358926

### Notes
This PR uses our existing WKEmptyView component for the empty state in image recommendations.

### Test Steps
1. On Staging, tap the Explore feed card to reach to the Image Recommendations feature.
2. Tap "Not sure" on recommendation 10 times to reach the end of the initial fetch list.
3. Confirm empty state appears.
4. Regression - confirm Watchlist empty states are unchanged.

### Screenshots/Videos
![IMG_0340](https://github.com/wikimedia/wikipedia-ios/assets/3620196/daf9cebf-80b7-4cc7-a564-9fc7d62c2921)
